src/views/Login.vue

```diff
@@ -21,6 +21,7 @@
               prepend-icon="mdi-account"
               type="text"
               v-model="user"
+              v-on:keyup.enter="handleSubmit"
```

Comment:
👍  could we maybe set this on the `v-form` element so we don't have to set it for every input field again?